cmd/liqoctl/cmd/network.go

```diff
@@ -165,6 +165,14 @@ func newNetworkConnectCommand(ctx context.Context, options *network.Options) *co
 	// Client flags
 	cmd.Flags().StringVar(&options.ClientGatewayType, "client-type", forge.DefaultGwClientType,
 		"Type of Gateway Client. Leave empty to use default Liqo implementation of WireGuard")
+	cmd.Flags().StringVar(&options.ClientConnectAddress, "client-address", "",
+		"Define the address used by the gateway client to connect to the gateway server."+
+			"This value overrides the one automatically retrieved by Liqo and it is useful when the server is "+
```

Comment:
```suggestion
		"Define the address used by the gateway client to connect to the gateway server. "+
			"This value overrides the one automatically retrieved by Liqo and it is useful when the server is "+
```
I think we are missing a space here, the same for the line below and for the peer command

Reply:
Fixed! Thank you